test/unit/audio-bandwidth-test.js

```diff
@@ -0,0 +1,351 @@
+import test from 'ava';
+import sinon from 'sinon';
+
+import AudioBandwidthTest from '../../src/diagnostics/AudioBandwidthTest';
+
+let audioBandwidthTest, options;
+test.before(() => {
```

Comment:
Does before run before each, or is there a separate beforeEach? Should we use that if so?